openwisp-config/files/openwisp.init

```diff
@@ -82,7 +86,8 @@ start_service() {
 	                        $bootup_delay $unmanaged $merge_config $test_config \
 	                        $test_script $connect_timeout $max_time $capath $cacert \
 	                        $mac_interface $management_interface $default_hostname \
-	                        $pre_reload_hook $post_reload_hook $post_registration_hook
+	                        $pre_reload_hook $post_reload_hook $post_reload_delay \
+	                        $post_registration_hook
```

Comment:
you forgot to add `test_retries` here

Reply:
Oops, I already had it there, but it got lost on the way...

Reply:
Added it again.